.rustfmt.toml

```diff
@@ -1 +1,3 @@
 hard_tabs = true
+group_imports = "StdExternalCrate"
+imports_granularity = "Crate"
```

Comment:
Could you add a newline while you are at it? 😅

Reply:
:pensive: